_nx_cugraph/__init__.py

```diff
@@ -232,6 +239,9 @@
         "eigenvector_centrality": {
             "dtype : dtype or None, optional": "The data type (np.float32, np.float64, or None) to use for the edge weights in the algorithm. If None, then dtype is determined by the edge values.",
         },
+        "forceatlas2_layout": {
+            "bool": "Distributes attraction along outbound edgesHubs attract less and thus are pushed to the borders.",
```

Comment:
I'm assuming "edgesHubs" wasn't intended?
```suggestion
            "bool": "Distributes attraction along outbound edges. Hubs attract less and thus are pushed to the borders.",
```